radicle/src/keystore.rs

```diff
@@ -18,6 +18,8 @@ pub enum Error {
     FromUtf8Error(#[from] std::string::FromUtf8Error),
     #[error("invalid key format: {0}")]
     Crypto(#[from] crypto::Error),
+    #[error("{0} not found")]
```

Comment:
I think it would make more sense to report what was missing. So more like:
```rust
#[error("profile keys did not exist at 'radicle.pub` or 'radicle'")]
KeysNotFound,
```

Reply:
Its the error being reported by the Profile::load() though.  Would we be good with wrapping the KeyNotFound error with NotFound(..).  Having a generic to signal when something doesn't exist is pretty useful, in place of returning an Option::None.

Reply:
Why put it in `keystore::Error` then? Maybe there should be a `profile::Error` that contains a variant for `keystore::Error` and `NotFound`. I also still think the error should be informative on how to fix the issue, `"profile not found"` is not very helpful :)

Reply:
> Why put it in `keystore::Error` then? Maybe there should be a `profile::Error` that contains a variant for `keystore::Error` and `NotFound`.

Done :+1:

> I also still think the error should be informative on how to fix the issue, `"profile not found"` is not very helpful :)

From which perspective?  User facing or developer facing?  I've elaborated the error by including the RAD_HOME path the profile was attempted to load from.

Reply:
Both :smile: It's good to have information for solving an issue whether you're a dev or a user -- and in Radicle's case a lot of users are devs :P